pkg/telemetery/settings.go

```diff
@@ -0,0 +1,13 @@
+package telemetery
+
+import (
+	"github.com/jaegertracing/jaeger/pkg/jtracer"
+	"go.opentelemetry.io/otel/metric"
+	"go.uber.org/zap"
+)
+
+type TelemeterySetting struct {
+	Logger        *zap.Logger
+	Tracer        *jtracer.JTracer
```

Comment:
I am not sure we still need this jtracer wrapper, it should just be tracer provider. jtracer should go away at the end of your refactoring.

Reply:
Noted, will do that.